tests/integration/conftest.py

```diff
@@ -261,3 +290,166 @@ def create_self_signed_tls_secret(host):
 
     for secret in created_secrets:
         kube_core_client.delete_namespaced_secret(name=secret, namespace=namespace)
+
+
+@pytest.fixture(scope="module", name="deploy_and_wait_for_mysql_pod")
+def deploy_and_wait_for_mysql_pod_fixture(ops_test, kube_core_client):
+    """Return an async function that deploy and wait for a mysql pod ready in current namespace."""
```

Comment:
I'd explain here why you need to deploy a "pod" manually, as while reading the code I was confused on why you'd need such fixture (until I saw the test)

Reply:
Added, thanks!